src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/Filtering/AccessibilityFilter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.ApiCompatibility.Abstractions
+{
+    internal class AccessibilityFilter : IDiffingFilter
```

Comment:
we should make these filters public.

Reply:
These is used based on settings/parameters that you pass down to the `ApiDiffer` so I don't think it makes much sense making it public at the moment, do you see a scenario where it could be used?